app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticViewModel.kt

```diff
@@ -0,0 +1,38 @@
+package com.fivesysdev.standwithukraine.ui
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.fivesysdev.standwithukraine.data.DayStatistic
+import com.fivesysdev.standwithukraine.data.DayStatisticRepositoryImpl
+import java.time.LocalDate
+
+class StatisticViewModel : ViewModel() {
+
+    private val dayStatistic = MutableLiveData<DayStatistic>()
+    private val dayStatisticRepository = DayStatisticRepositoryImpl()
+    var date: LocalDate = LocalDate.now()
+        private set
+
+    init {
+        setDayStatistic()
+    }
+
+    fun getCurrentDayStatistic(): DayStatistic? {
+        return dayStatistic.value
```

Comment:
Why don't you use live data with an observer instead of this?

Reply:
I don't know, I was thinking about the easiest way. I can fix it. In ViewModel:
`fun getDayStatistic() : LiveData<DayStatistic> {
        return dayStatistic
    }`
In Fragment: 
`viewModel.getDayStatistic().observe(viewLifecycleOwner) {
            setDayStatistic(it)
        }`
Is it correct?

Reply:
the usual definition is: 
```
private val _dayStatistic = MutableLiveData<DayStatistic?>()
val dayStatistic: LiveData<DayStatistic?>
        get() = _dayStatistic
```

Reply:
Thanks. I understood. Is there something else wrong?